main/lsp/LSPIndexer.cc

```diff
@@ -10,86 +10,24 @@
 using namespace std;
 
 namespace sorbet::realmain::lsp {
-
 namespace {
-
-const core::FileHash &findHash(int id, const vector<core::FileHash> &globalStateHashes,
```

Comment:
Hashes are now stored in files, so the new `getOldFile` method accomplishes the same thing.